src/data/pull_requests.json.js

```diff
@@ -61,6 +62,10 @@ const { repository } = await octokit.graphql.paginate(
   }`,{
       organization: gh_organization,
       repo: gh_repo,
+      // I trim the branch value because, make (-include .env) forward env variable with \n, 
+      // I spend times to understand why it does not work when doing make download, 
+      // but is working when invoking the script directly
```

Comment:
hmm weird bug, but thanks for the investigation.
Maybe we will find a better fix later. I keep this one for now

Perhaps there is a different behavior with dotenv also, that makes it work for direct invok